### ISSUE TITLE
Open in a new tab

### ISSUE THREAD

**Issue by A-d-r-i** [tankaru/OpenSwitchMaps#53]
I use this extension a lot on Firefox (88.0.1) and just ran into a problem. I can't switch to another card in a new tab (with a middle click).
It would be perhaps interesting to incorporate a function allowing to activate (or not) by default the opening in a new tab.

Thank you in advance for your help!

**Comment by Dimitar5555** [comment]
Currently it's possible by clicking with the middle button but the opening in a new tab is blocked by Firefox.

**Comment by pigsonthewing** [comment]
> Currently it's possible by clicking with the middle button 

Not all devices (example: my Dell XPS laptop) have a middle button (or equivalent function).

> but the opening in a new tab is blocked by Firefox.

AIUI this can be overridden locally.


Please add an "always open in new tab" preference setting.

**Comment by tankaru** [comment]
> Please add an "always open in new tab" preference setting.
I'll try.

**Comment by tankaru** [comment]
I've add an "always open in new tab" preference setting on version 1.5.0.

**Comment by Cyrille37** [comment]
related to #135
